src/Command/ClearCommand.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Webgriffe\SyliusAkeneoPlugin\Command;
+
+use DateInterval;
+use DateTime;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\Persistence\ManagerRegistry;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Webgriffe\SyliusAkeneoPlugin\Entity\QueueItem;
+use Webgriffe\SyliusAkeneoPlugin\Repository\QueueItemRepositoryInterface;
+
+final class ClearCommand extends Command
+{
+    public const SUCCESS = 0;
+
+    public const FAILURE = 1;
+
+    public const DEFAULT_DAYS = 10;
+
+    public const DAYS_OPTION_NAME = 'days';
+
+    /** @var QueueItemRepositoryInterface */
+    private $queueItemRepository;
+
+    /** @var ManagerRegistry */
+    private $managerRegistry;
+
+    // the name of the command (the part after "bin/console")
+    protected static $defaultName = 'webgriffe:akeneo:clear';
+
+    /**
+     * ClearCommand constructor.
+     */
+    public function __construct(QueueItemRepositoryInterface $queueItemRepository, ManagerRegistry $managerRegistry)
+    {
+        $this->queueItemRepository = $queueItemRepository;
+        $this->managerRegistry = $managerRegistry;
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setDescription('Clean the Akeneo\'s queue of items older than N days.')
+            ->setHelp('This command allows you to clean the Akeneo\'s queue of item older than a specificed numbers of days.')
+            ->addOption(
```

Comment:
I'd use a required argument and not an option. The number of days should be always provided.